labcore/analysis/hvapps.py

```diff
@@ -289,7 +293,7 @@ class LoaderNodeBase(Node):
     Each subclass must implement ``LoaderNodeBase.load_data``.
     """
 
-    def __init__(self, *args: Any, **kwargs: Any):
+    def __init__(self, path, *args: Any, **kwargs: Any):
```

Comment:
Add a default here, make the default just be `.` if you can't find something. I don't like suddenly changing the signature and making it mandatory.